### DISCUSSION TITLE
Use FlareSolverr as a real proxy

### DISCUSSION THREAD

**Discussion by mk7120** [FlareSolverr/FlareSolverr#648] [Q&A]
Is there anyway that I can use FlareSolverr as a real proxy when sending a request?

For example, I would like:
`curl --proxy "http://localhost:8191" "https://www.google.com"`

instead of

```
curl -L -X POST 'http://localhost:8191/v1' \
-H 'Content-Type: application/json' \
--data-raw '{
  "cmd": "request.get",
  "url":"http://www.google.com/",
  "maxTimeout": 60000
}'
```

**Comment by ngosang** [comment]
This is not supported. Someone with technical skill could write a proxy integrated with FlareSolverr.

**Reply by S0me1-dev** [reply]
Hello, I am too interested to use flaresolverr as a proxy. 

for my browser , or parser or anything else
With proxifier maybe or idk. But flaresolverr I think don't understand requests in all formats. Plz make it function, or to catch maybe whole traffic function. 
Guys plz help.

There is chatgpt now, but I can't logically tell him what to do , cause don't know how to provide that function, and what exactly need to be coded, added

**Reply by SecT0uch** [reply]
At first I really though Flaresolverr was exposing a standard HTTP/SOCKS proxy, so we could use it with any service that support HTTP proxies. 
@ngosang  I wonder why the choice was made to make custom requests to use Flaresolverr ?
Is there any limitation/much extra work ?
Maybe we could Pin this as an issue with "help-required" tag for more visibility ?
